src/luasql.c

```diff
@@ -131,3 +131,33 @@ LUASQL_API void luasql_set_info (lua_State *L) {
 	lua_pushliteral (L, "LuaSQL 2.3.0");
 	lua_settable (L, -3);
 }
+
+/*
+** Pulls an optional string value from the table at idx
+*/
+LUASQL_API const char* luasql_table_optstring(lua_State *L, int idx, const char* name, const char* def) {
+	const char* res = NULL;
+
+	lua_pushstring(L, name);
+	lua_gettable(L, idx);
+
+	res = lua_tostring(L, -1);
+	lua_pop(L, 1);
+
+	return (res != NULL) ? res : def;
+}
+
+/*
+** Pulls an optional number value from the table at idx
+*/
+LUASQL_API lua_Number luasql_table_optnumber(lua_State *L, int idx, const char* name, lua_Number def) {
+	lua_Number res = 0;
+
+	lua_pushstring(L, name);
+	lua_gettable(L, idx);

```

Comment:
The last person who I saw keeping Lua 5.0 compatibility in any library was @tomasguisasola. Since he's maintaining LuaSQL drivers for his uses in production, he'd be the best person to ask. Tomás, do you still need 5.0 compatibility for anything?